### DISCUSSION TITLE
GoDaddy - faild to get zone: could not find the start of authority for _acme-challenge.example.com.: SERVFAIL

### DISCUSSION THREAD

**Discussion by SquallHalle** [go-acme/lego#1575] [Q&A] [answered]
### Welcome

- [X] Yes, I'm using a binary release within 2 latest releases.
- [X] Yes, I've searched similar issues on GitHub and didn't find any.
- [X] Yes, I've included all information below (version, config, etc).

### What did you expect to see?

successful optaining a certificate

### What did you see instead?

acme: error presenting token: godaddy: failed to get zone: could not find the start of authority for _acme-challenge.example.com.: NOERROR

### How do you use lego?

Binary

### Reproduction steps

1. run alpine
2. apk add lego
3. export GODADDY_API_SECRET=SECRET
4. export GODADDY_API_KEY=KEY
5. lego --email mail@example --dns godaddy --domains example.com run

### Version of lego

```console
lego version 4.1.3 linux/amd64
```

### Logs

<details>

```
# lego --email mail@example --dns godaddy --domains example.com run

2022/01/29 20:51:18 [INFO] [example.com] acme: Obtaining bundled SAN certificate
2022/01/29 20:51:19 [INFO] [example.com] AuthURL: https://acme-v02.api.letsencrypt.org/acme/authz-v3/73202870480
2022/01/29 20:51:19 [INFO] [example.com] acme: Could not find solver for: tls-alpn-01
2022/01/29 20:51:19 [INFO] [example.com] acme: Could not find solver for: http-01
2022/01/29 20:51:19 [INFO] [example.com] acme: use dns-01 solver
2022/01/29 20:51:19 [INFO] [example.com] acme: Preparing to solve DNS-01
2022/01/29 20:51:19 [INFO] [example.com] acme: Cleaning DNS-01 challenge
2022/01/29 20:51:19 [WARN] [example.com] acme: cleaning up failed: godaddy: failed to get zone: could not find the start of authority for _acme-challenge.example.com.: SERVFAIL
2022/01/29 20:51:19 [INFO] Deactivating auth: https://acme-v02.api.letsencrypt.org/acme/authz-v3/73202870480
2022/01/29 20:51:19 Could not obtain certificates:
        error: one or more domains had a problem:
[example.com] [example.com] acme: error presenting token: godaddy: failed to get zone: could not find the start of authority for _acme-challenge.example.com.: SERVFAIL
```

</details>

**Comment by SquallHalle** [comment]
if you see this as a question please move the lable then :-)

**Comment by ldez** [comment]
Hello,

it's a problem with your network, the DNS queries made by lego fail because your DNS doesn't seem to respond as expected.

It can be related to a local DNS, a firewall, a network configuration, etc.

So you have to check your network.

You can also try adding resolvers ([`--dns.resolvers`](https://go-acme.github.io/lego/usage/cli/)).
ex:
```
lego  --dns.resolvers 8.8.8.8 --email mail@example --dns godaddy --domains example.com run
```

**Comment by SquallHalle** [comment, marked as answer]
if i use DIG there is no answer section on my local network.
onlinetools show me answer sections.
I use OpenWrt and DNS HTTPS Proxy Settings for DNS OVER HTTPS.
There is an Option: Force Router DNS with the values "Let local devices use their own DNS servers if set" and "Force Router DNS server to all local devices"
You can guess that "Force Router DNS server to all local devices" was default and filtered the answersection.
Weired ! 

Its working now thank you guys !
